pymc_bart/pgbart.py

```diff
@@ -488,6 +503,20 @@ def fast_mean(ari):
         return res / count
 
 
+@njit
+def fast_linear_fit(x, y):
+    """Use Numba to speed up the computation of the linear fit"""
+    n = len(x)
+    xbar = np.sum(x) / n
+    ybar = np.sum(y) / n
+
+    b = (x @ y.T - n * xbar * ybar) / (x @ x - n * xbar**2)
+    a = ybar - b * xbar
+
+    y_fit = a + b * x
+    return y_fit, [a, b]
```

Comment:
Not sure about this, but probably we want

```suggestion
    return y_fit, [a.item(), b.item()]
```

Reply:
added in https://github.com/pymc-devs/pymc-bart/pull/79/commits/996596f2053bd4bb1b942a5b483d712d84b042c0